login.html

```diff
@@ -50,10 +50,10 @@ <h1 class="header-logo">Beads & Beyonds....</h1>
     
                 </div>
                 <div class="avatar centered mb-lg">
-                    <img class="border-round img-border md-img" src="./assets/toy1.jpg" alt="Avatar">
+                    <img class="border-round img-border img-md" src="./assets/toy1.jpg" alt="Avatar">
```

Comment:
suggestion: the class names can be better arranged, class="img-md img-border border-round" if its not messing up with your specificity or cascading.